vsintegration/tests/UnitTests/InlineParameterNameHintTests.fs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace VisualFSharp.UnitTests.Editor.Hints
+
+open NUnit.Framework
+open HintTestFramework
+
+module InlineParameterNameHintTests =
+
```

Comment:
Maybe add a test for an operator. Built-in and user defined.

Reply:
Add a test, will comment on operators in another thread